src/main/resources/ready-api-rest-api-spec.json

```diff
@@ -327,6 +327,92 @@
       }
     }
   },
+  "/v1/readyapi/executions/{executionID}/logs": {
```

Comment:
Sure, one problem I have is that sometimes we need to work on different branches (specially when multuple people are working on it) and the SwaggerHub definition is compatible only with  one of them. But probably we can keep a short lived local copy of the SwaggerHub definition.